### ISSUE TITLE
systemd seems to be failing to compile on CentOS

### ISSUE THREAD

**Issue by evverx** [systemd/systemd#11036]
```
[148/1733] Compiling C object 'src/libsystemd/src@libsystemd@@systemd_static@sta/sd-netlink_netlink-types.c.o'.
FAILED: src/libsystemd/src@libsystemd@@systemd_static@sta/sd-netlink_netlink-types.c.o
cc -Isrc/libsystemd/src@libsystemd@@systemd_static@sta -Isrc/libsystemd -I../src/libsystemd -Isrc/basic -I../src/basic -Isrc/shared -I../src/shared -Isrc/systemd -I../src/systemd -Isrc/journal -I../src/journal -Isrc/journal-remote -I../src/journal-remote -Isrc/nspawn -I../src/nspawn -Isrc/resolve -I../src/resolve -Isrc/timesync -I../src/timesync -I../src/time-wait-sync -Isrc/login -I../src/login -Isrc/udev -I../src/udev -Isrc/libudev -I../src/libudev -Isrc/core -I../src/core -I../src/libsystemd/sd-bus -I../src/libsystemd/sd-device -I../src/libsystemd/sd-event -I../src/libsystemd/sd-hwdb -I../src/libsystemd/sd-id128 -I../src/libsystemd/sd-netlink -I../src/libsystemd/sd-network -Isrc/libsystemd-network -I../src/libsystemd-network -I. -I../ -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=gnu99 -g -Wextra -Werror=undef -Wlogical-op -Wmissing-include-dirs -Wold-style-definition -Wpointer-arith -Winit-self -Wfloat-equal -Wsuggest-attribute=noreturn -Werror=missing-prototypes -Werror=implicit-function-declaration -Werror=missing-declarations -Werror=return-type -Werror=format=2 -Wstrict-prototypes -Wredundant-decls -Wmissing-noreturn -Wshadow -Wendif-labels -Wstrict-aliasing=2 -Wwrite-strings -Werror=overflow -Wnested-externs -Wno-unused-parameter -Wno-missing-field-initializers -Wno-unused-result -Wno-error=nonnull -Wno-maybe-uninitialized -ffast-math -fno-common -fdiagnostics-show-option -fno-strict-aliasing -fvisibility=hidden -fstack-protector -fstack-protector-strong --param=ssp-buffer-size=4 -fPIE -Werror=shadow -include config.h -g -O0 -ftrapv -fPIC -pthread -fvisibility=default  -MD -MQ 'src/libsystemd/src@libsystemd@@systemd_static@sta/sd-netlink_netlink-types.c.o' -MF 'src/libsystemd/src@libsystemd@@systemd_static@sta/sd-netlink_netlink-types.c.o.d' -o 'src/libsystemd/src@libsystemd@@systemd_static@sta/sd-netlink_netlink-types.c.o' -c ../src/libsystemd/sd-netlink/netlink-types.c
../src/libsystemd/sd-netlink/netlink-types.c:636:10: error: ‘FRA_SUPPRESS_IFGROUP’ undeclared here (not in a function)
         [FRA_SUPPRESS_IFGROUP]    = { .type = NETLINK_TYPE_U32 },
          ^
../src/libsystemd/sd-netlink/netlink-types.c:636:9: error: array index in initializer not of integer type
         [FRA_SUPPRESS_IFGROUP]    = { .type = NETLINK_TYPE_U32 },
         ^
../src/libsystemd/sd-netlink/netlink-types.c:636:9: error: (near initialization for ‘rtnl_routing_policy_rule_types’)
../src/libsystemd/sd-netlink/netlink-types.c:637:10: error: ‘FRA_SUPPRESS_PREFIXLEN’ undeclared here (not in a function)
         [FRA_SUPPRESS_PREFIXLEN]  = { .type = NETLINK_TYPE_U32 },
          ^
../src/libsystemd/sd-netlink/netlink-types.c:637:9: error: array index in initializer not of integer type
         [FRA_SUPPRESS_PREFIXLEN]  = { .type = NETLINK_TYPE_U32 },
         ^
../src/libsystemd/sd-netlink/netlink-types.c:637:9: error: (near initialization for ‘rtnl_routing_policy_rule_types’)
[149/1733] Compiling C object 'src/libsystemd/src@libsystemd@@systemd_static@sta/sd-netlink_netlink-util.c.o'.
ninja: build stopped: subcommand failed.
```

**Comment by evverx** [comment]
To judge from https://github.com/systemd/systemd-centos-ci/issues/18#issuecomment-442400846, everything was more or less fine 5 days ago.

**Comment by evverx** [comment]
I "fixed" it by reverting 9714c020fc4cda1823c2a77e3f. @yuwata could you take a look?
